<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SkypePhone</class>
 <widget class="QWidget" name="SkypePhone">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>865</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>vvvvvvvvvvvv</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>使用Skype公共API连接到Skype客户端</string>
       </property>
       <property name="text">
        <string>连接Skype</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>当前登陆的Skype用户名</string>
       </property>
       <property name="text">
        <string>未检测到</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_4">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>开始呼叫电话</string>
       </property>
       <property name="text">
        <string>呼叫</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QComboBox" name="comboBox_3">
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>9086013567890</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QToolButton" name="toolButton_2">
       <property name="toolTip">
        <string>显示/隐藏拨号盘</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="kp.qrc">
         <normaloff>:/skins/default/keyboard.png</normaloff>:/skins/default/keyboard.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_14">
       <property name="toolTip">
        <string>Skype客户端状态</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="kp.qrc">:/skins/default/status_offline.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="4">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="2">
      <widget class="QToolButton" name="toolButton_14">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="kp.qrc">
         <normaloff>:/skins/default/digit0.png</normaloff>:/skins/default/digit0.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QToolButton" name="toolButton_15">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="kp.qrc">
         <normaloff>:/skins/default/digit1.png</normaloff>:/skins/default/digit1.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QToolButton" name="toolButton_16">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="kp.qrc">
         <normaloff>:/skins/default/digit2.png</normaloff>:/skins/default/digit2.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QToolButton" name="toolButton_17">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="kp.qrc">
         <normaloff>:/skins/default/digit3.png</normaloff>:/skins/default/digit3.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QToolButton" name="toolButton_18">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="kp.qrc">
         <normaloff>:/skins/default/digit4.png</normaloff>:/skins/default/digit4.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QToolButton" name="toolButton_19">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="kp.qrc">
         <normaloff>:/skins/default/digit5.png</normaloff>:/skins/default/digit5.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QToolButton" name="toolButton_20">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="kp.qrc">
         <normaloff>:/skins/default/digit6.png</normaloff>:/skins/default/digit6.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QToolButton" name="toolButton_21">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="kp.qrc">
         <normaloff>:/skins/default/digit7.png</normaloff>:/skins/default/digit7.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QToolButton" name="toolButton_22">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="kp.qrc">
         <normaloff>:/skins/default/digit8.png</normaloff>:/skins/default/digit8.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QToolButton" name="toolButton_23">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="kp.qrc">
         <normaloff>:/skins/default/digit9.png</normaloff>:/skins/default/digit9.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QToolButton" name="toolButton_24">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="kp.qrc">
         <normaloff>:/skins/default/digitstar.png</normaloff>:/skins/default/digitstar.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QToolButton" name="toolButton_25">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="kp.qrc">
         <normaloff>:/skins/default/digitpound.png</normaloff>:/skins/default/digitpound.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="kp.qrc">:/skins/default/speaker.png</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>call status message here.................</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="kp.qrc">:/skins/default/Un-mute_Alpha_down_16x16_s1.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>三</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="maximumSize">
          <size>
           <width>35</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="kp.qrc">:/skins/default/user.png</pixmap>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>99008668056013552776960</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Others</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>三</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QToolButton" name="toolButton_12">
         <property name="toolTip">
          <string>开启视频</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="kp.qrc">
           <normaloff>:/skins/default/Call_video_start_GreenStart_32x32_s1.png</normaloff>:/skins/default/Call_video_start_GreenStart_32x32_s1.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_10">
         <property name="toolTip">
          <string>挂断通话</string>
         </property>
         <property name="text">
          <string>挂断</string>
         </property>
         <property name="icon">
          <iconset resource="kp.qrc">
           <normaloff>:/skins/default/hangup_20.png</normaloff>:/skins/default/hangup_20.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="icon">
       <iconset resource="kp.qrc">
        <normaloff>:/skins/default/book_32.png</normaloff>:/skins/default/book_32.png</iconset>
      </attribute>
      <attribute name="title">
       <string>联系人</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QToolButton" name="toolButton_6">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_8">
           <property name="editable">
            <bool>true</bool>
           </property>
           <item>
            <property name="text">
             <string>新建项目</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>新建项目</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>新建项目</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>新建项目</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>新建项目</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_7">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_8">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeView" name="treeView"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>呼叫历史</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QComboBox" name="comboBox_9">
           <property name="editable">
            <bool>true</bool>
           </property>
           <item>
            <property name="text">
             <string>新建项目</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>新建项目</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>新建项目</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>新建项目</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>新建项目</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_10">
           <item>
            <property name="text">
             <string>All</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>新建项目</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>新建项目</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>新建项目</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>新建项目</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_9">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="tableView"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="kp.qrc"/>
 </resources>
 <connections/>
</ui>
